README.md

```diff
@@ -22,13 +22,55 @@ Highlights:
 * Easy to configure with GUIs and APIs.
 
 We walk on the shoulders of giants; Opstrace contains all the open source projects you know and love in one place:
+
 * [Cortex](https://github.com/cortexproject/cortex)
 * [Loki](https://github.com/grafana/loki)
 * [Prometheus](https://github.com/prometheus/prometheus)
 * [Kubernetes](https://github.com/kubernetes/kubernetes)
 * and many more!
 
 
+## Key Features
+
+### Installation and Upgrades
+
+Both installation and upgrades are initiated by a single CLI command, and orchestrated behind the scenes.
+The installer users your local credentials to setup the necessary cloud resources (e.g., a EKS or GKE cluster) and then deploys the Opstrace controller which orchestrates all of the app-level deployments.
+
+```bash
+$ cat << EOF > config.yaml
+ tenants:
+  - staging
+  - prod
+EOF
```

Comment:
this code block comes as a surprise

Reply:
How so?  This is very similar to what we have in the quick start, but pared down so it is as short as possible.